src/cmd/vm.rs

```diff
@@ -379,18 +387,42 @@ fn run_acloudw_cmd(opts: &[&str]) -> Result<()> {
     let acloudw_cmd = opts.join(" ");
     info!("Running `{acloudw_cmd}`...");
     let mut cmd = Command::new("bash")
+        .stdout(Stdio::piped())
         .arg("-c")
         .arg(acloudw_cmd)
         .spawn()
         .context("Failed to execute acloudw")?;
 
+    let buf_reader = BufReader::new(
+        cmd.stdout
+            .take()
+            .context("Failed to get stdout from output")?,
+    );
+    let port = get_acloudw_port_number(buf_reader)?;
+
     let result = cmd.wait().context("Failed to wait for acloudw")?;
 
     if !result.success() {
         error!("acloudw failed")
     }
 
-    Ok(())
+    Ok(port)
+}
+
+fn get_acloudw_port_number(r: impl BufRead) -> Result<String> {
```

Comment:
Also please add a function-level documentation about the return value.

Reply:
Added a comment!